src/main/java/de/fearnixx/jeak/service/token/ITokenService.java

```diff
@@ -1,13 +1,27 @@
 package de.fearnixx.jeak.service.token;
 
+import java.util.Set;
+
 public interface ITokenService {
     /**
      * Verify a given token.
+     * IMPORTANT: Only use this method after the {@link de.fearnixx.jeak.event.bot.IBotStateEvent.IPreInitializeEvent}
+     * was fired.
```

Comment:
This note is not really necessary. Only stuff like service creation and registration should be done in ``IPreInitializeEvent`` which means that no mention and the exception will suffice to show that the invokation was wayy to early.  
If it can be expected that this method would be called during ``IPreInitialize``, use the ``order`` property on the corresponding pre-init listener.